src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const getPartnerLabel = () => {
+    let marriedLabel;
+
+    if (isMarried) {
+      marriedLabel = sex === 'm'
+        ? `${partnerName} is my wife`
+        : `${partnerName} is my husband`;
+    }
+
+    return isMarried === true
+      ? marriedLabel
+      : 'I am not married';
+  };
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age !== undefined
```

Comment:
```suggestion
      {!!age
```
you may just transform it into boolean - FYI